packages/mds-policy-service/tests/policy-client.spec.ts

```diff
@@ -0,0 +1,322 @@
+/* eslint-disable no-console */
+import { GeographyServiceClient, GeographyServiceManager } from '@mds-core/mds-geography-service'
+import {
+  POLICY2_JSON,
+  POLICY3_JSON,
+  POLICY_WITH_DUPE_RULE,
+  START_ONE_MONTH_AGO,
+  START_ONE_MONTH_FROM_NOW
+} from '@mds-core/mds-test-data'
+import { clone, days, now, uuid, yesterday } from '@mds-core/mds-utils'
+import { PolicyMetadataDomainModel } from '../@types'
+import { PolicyServiceClient } from '../client'
+import { PolicyRepository } from '../repository'
+import { PolicyServiceManager } from '../service/manager'
+import { GeographyFactory, PolicyFactory, RulesFactory } from './helpers'
+
+const GeographyServer = GeographyServiceManager.controller()
+const PolicyServer = PolicyServiceManager.controller()
+
+describe('spot check unit test policy functions with SimplePolicy', () => {
+  describe('Policy Client Tests', () => {
+    beforeAll(async () => {
+      await GeographyServer.start()
+      await PolicyServer.start()
+      await PolicyRepository.initialize()
+      // await GeographyServiceClient.writeGeographies([VENICE_GEOGRAPHY])
```

Comment:
nit: delete

Reply:
Was going to comment this, then refreshed browser, and I see I was beaten to it.